### PR TITLE
RC v0.0.10

### PR DESCRIPTION
**Pull request by aburrell**
# Description

A bugfix release to ensure preliminary F10.7 data is downloaded when needed.

# Type of change

- Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

```
import datetime as dt
import pysat

stime = # Set to a date to past the last partial quarterly file you have, will vary by user
f107 = pysat.Instrument('sw', 'f107', tag='prelim', use_header=True)
f107.dowload(start=stime)
f107.files.files[stime]
```

In the old version, if you had previously downloaded a quarterly file and it needed to be updated, the update wouldn't happen.

## Test Configuration
* Operating system: OS X Big Sur
* Version number: Python 3.8
* Any details about your local setup that are relevant: main version of pysat

# Checklist:

- [x] Update the version number
- [x] Verify and update Zenodo information in .zenodo.json
- [x] doc tests pass locally
- [x] activate this branch on readthedocs (see link in CI)
- [x] Upload the test distribution to the testPyPi server: https://test.pypi.org/project/pysatSpaceWeather/0.0.10/
- [x] Contacted developers and users about the future release
- [x] Performed integration tests with downstream packages (include list of these packages)
- [x] Create a wiki page with the release text (https://github.com/pysat/pysatSpaceWeather/wiki/v0.0.10)
- [x] After approval, confirm all changes from the RC pull are listed in the wiki release text.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes

If this is a release PR, replace the first item of the above checklist with the
release checklist on the pysat wiki:
https://github.com/pysat/pysat/wiki/Checklist-for-Release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,12 @@ Change Log
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](https://semver.org/).
 
+[0.0.10] - 2023-04-18
+---------------------
+* Bugs
```
Comment (aburrell) [inline CHANGELOG.md:8]:
```suggestion
---------------------
* Maintenance
  * Bumped the NEP29 numpy version in tests
* Bugs
```

**Review by jklenzing — APPROVED** [review]
Looks good!

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,18 @@ Change Log
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](https://semver.org/).
 
+[0.0.10] - 2023-05-05
```
Comment (aburrell) [inline CHANGELOG.md:6]:
```suggestion
[0.0.10] - 2023-05-19
```

**Comment by aburrell** [comment]
@rstoneback @JonathonMSmith can I get one of you to review this today?  Otherwise the release will get pushed further back since I'll be out.

**Review by rstoneback — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,18 @@ Change Log
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](https://semver.org/).
 
+[0.0.10] - 2023-05-19
```
Comment (aburrell) [inline CHANGELOG.md:6]:
```suggestion
[0.0.10] - 2023-06-01
```

**Pull request merged**